react/AppIcon/Preloader.js

```diff
@@ -0,0 +1,58 @@
+const _loaded = {}
```

Comment:
I am not a fan of the Preloader name, for me it is just a cache.

`Preloader.getPreloaded` -> `cache.get`
`Preloader.preload` -> `cache.fetch`

Reply:
Actually the pattern is an image preloading.

Reply:
Yes I knew the name :) I still prefer the names cache / get and fetch, I find `getPreloaded` a bit ugly :)